lib/tulul_stats_bot/tulul_stats/group.rb

```diff
@@ -80,6 +80,12 @@ def top(field)
 
         ret += '</pre>'
         ret
+      elsif field.downcase == 'last_tulul'
+        rank = 0
+        self.users.reject{|b| b.last_tulul_at.nil?}.sort_by{|b| b.last_tulul_at}.inject('') do |prev, cur|
```

Comment:
`reject{ |a| do_something(a) }`? 

`map.join("\n")` instead of `inject(''){ "\n" }`?